std/xml.d

```diff
@@ -2970,3 +2982,17 @@ private
         throw new XMLException(s);
     }
 }
+
+unittest
+{
+    string test_xml = `<?xml version="1.0" encoding='UTF-8'?><stream:stream xmlns:stream="http://etherx.'jabber'.org/streams" xmlns="jabber:'client'" from='jid.pl' id="587a5767" xml:lang="en" version="1.0"></stream:stream>`;
+
+    try
+    {
+        DocumentParser parser = new DocumentParser(test_xml);
+    }
+    catch (Exception e)
+    {
+        assert(1, e.toString());
+    }
```

Comment:
I think you don't need the try-catch-assert thing here. First, `assert(1);` always passes. I think you want `assert(0, ...);` - so that an `AssertError` is thrown instead of a `TagException`. I don't think you need to do that, though. An uncaught exception in a `unittest` block should be ok as a failure signal. I'm not sure though.